lib/views/after_auth_screens/profile/profile_page.dart

```diff
@@ -204,20 +243,7 @@ class ProfilePage extends StatelessWidget {
                             const Tab(text: 'Tasks'),
                           ],
                           views: [
-                            ColoredBox(
-                              color: Theme.of(context).colorScheme.background,
-                              child: GridView.count(
-                                mainAxisSpacing: 5,
-                                crossAxisCount: 3,
-                                children: [
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                ],
-                              ),
-                            ),
+                            IndividualPostView(post: samplePost),
```

Comment:
Write the test for this too. To get this file to 100%

Reply:
@noman2002 I have used this widget directly from the file individual_view.dart. The tests for it are already there in individual_view_test.dart and all test have passed successfully. I have already uploaded the screenshots for reference also in previous conversations. Kindly tell me if any other issue exists. The code is also working fine, I have provided the screenshots for this also above.

Reply:
> @noman2002 I have used this widget directly from the file individual_view.dart. The tests for it are already there in individual_view_test.dart and all test have passed successfully. I have already uploaded the screenshots for reference also in previous conversations. Kindly tell me if any other issue exists. The code is also working fine, I have provided the screenshots for this also above.

Test for that widget is already used but we also need to get the coverage of this file to 100%. just write the test for it. It will be a simple 2 3 lines of code.

Reply:
@noman2002 Actually I have made the changes in the file profile_page.dart and there is no previously written test file for it. So can you please guide me where should i write test for this widget as no test file exists in the tests folder for this file.

Reply:
Create a file as per the current file structure of tests.

Reply:
@noman2002 I have written the test file as you suggested. Here is the screenshot showing all tests passed for profile_page.dart file - 
<img width="526" alt="image" src="https://github.com/PalisadoesFoundation/talawa/assets/125534462/70d4d32c-6a6c-4999-827d-ddcd2f35727b">